Tests/SentryTests/SentrySDKTests.swift

```diff
@@ -397,6 +397,58 @@ class SentrySDKTests: XCTestCase {
 
         assertHubScopeNotChanged()
     }
+    
+    // MARK: - Logger Flush Tests
+    
+    func testFlush_CallsLoggerFlush() {
+        fixture.client.options.experimental.enableLogs = true
+        SentrySDKInternal.setCurrentHub(fixture.hub)
+        SentrySDKInternal.setStart(with: fixture.client.options)
+        
+        // Add a log to ensure there's something to flush
+        SentrySDK.logger.info("Test log message")
+        
+        // Initially no logs should be sent (they're buffered)
+        XCTAssertEqual(fixture.client.captureLogsDataInvocations.count, 0)
+        
+        // Flush the SDK
+        SentrySDK.flush(timeout: 1.0)
+        
+        // Wait a bit for the log to be flushed
+        let expectation = self.expectation(description: "Wait for async add.")
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+            expectation.fulfill()
+        }
+        waitForExpectations(timeout: 0.5)
```

Comment:
`m`: Please use a way higher value, as we continuously see low timeouts for expectations time out in CI.
```suggestion
        waitForExpectations(timeout: 5.0)
```